js/__fixtures__/contexts/did-v1.json

```diff
@@ -0,0 +1,87 @@
+{
+    "@context": {
+      "@version": 1.1,
+      "id": "@id",
+      "type": "@type",
+      "dc": "http://purl.org/dc/terms/",
+      "schema": "http://schema.org/",
+      "sec": "https://w3id.org/security#",
+      "didv": "https://w3id.org/did#",
+      "xsd": "http://www.w3.org/2001/XMLSchema#",
+      "EcdsaSecp256k1VerificationKey2019": "sec:EcdsaSecp256k1VerificationKey2019",
```

Comment:
This is going to cause a redefinition error once the protected is fixed. These should be dropped from here and a reference to the v3 context should be included as well here in my opinion.

Reply:
this comment belongs on "did spec registries" :) I am just copying what is in https://github.com/w3c/did-spec-registries/blob/master/contexts/did-v1.jsonld

Reply:
I agree btw, but a similar comment to the vc data model applies, both W3C DID and W3C VC specs each have their own context files, and I suspect we won't get them to agree to resolving an unstable context in maintained by the CCG.

Reply:
I removed this